Xamarin.Forms.Platform.Tizen/ExportRendererAttribute.cs

```diff
@@ -1,12 +1,16 @@
 using System;
+using Xamarin.Forms.Core;
+using Information = Tizen.System.Information;
```

Comment:
Not sure why, but `Tizen` isn't available for me (even after installing the SDK) unless I short circuit class path like this.